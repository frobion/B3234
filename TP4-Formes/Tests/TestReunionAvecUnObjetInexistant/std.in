S Segment1 0 0 1 1
R Rectangle1 1 3 6 0
OR Reunion1 Segment1 Segment2 Rectangle1
LIST
EXIT
