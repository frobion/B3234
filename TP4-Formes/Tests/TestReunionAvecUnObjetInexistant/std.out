OK
OK
ERR
# Segment2 inexistant
R Rectangle1 1 3 6 0
S Segment1 0 0 1 1
